rotkehlchen/chain/evm/decoding/spark/savings/decoder.py

```diff
@@ -121,9 +122,12 @@ def _decode_spark_tokens_deposit_withdrawal(self, context: DecoderContext) -> De
         ):
             return DEFAULT_DECODING_OUTPUT
 
-        vault_token = self.base.get_or_create_evm_token(
-            address=context.tx_log.address,
-            encounter=(encounter := TokenEncounterInfo(should_notify=False)),
+        GlobalDBHandler.set_tokens_protocol_if_missing(
```

Comment:
you can pass `protocol=CPT_SPARK` and that way you don't need to call set_tokens_protocol_if_missing. Check how it works